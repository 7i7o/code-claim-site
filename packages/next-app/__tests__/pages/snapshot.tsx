import React from "react";
import renderer from "react-test-renderer";
import { ChakraProvider } from "@chakra-ui/react";

import Index from "@/pages/index";
import { theme } from "@/chakra.config";

it("renders homepage unchanged", () => {
  const tree = renderer
    .create(
      <ChakraProvider theme={theme}>
        <Index />
      </ChakraProvider>
    )
    .toJSON();
  expect(tree).toMatchSnapshot();
});
