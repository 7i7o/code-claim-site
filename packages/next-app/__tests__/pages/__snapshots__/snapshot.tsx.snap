// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders homepage unchanged 1`] = `
<div
  className="css-0"
>
  <div
    className="chakra-container css-0"
  >
    TODO
  </div>
</div>
`;
